scripts/src/view/input/base.tsx

```diff
@@ -55,11 +56,10 @@ namespace fugazi.view.input {
 		public constructor(props: P, className: string, prompt?: string) {
 			super(props);
 
-			this.state.value = props.value || "";
-
 			this.prompt = prompt;
 			this.className = ["input", className];
 			this.keymap = collections.map<() => void>();
+			this.state = {value: this.props.value || ""} as any;
```

Comment:
That's weird, I don't get errors for this.  Maybe due to the new ts version?
Fine, after the PR I'll take this branch and will try to figure it out.